p2p/peer.py

```diff
@@ -333,6 +335,9 @@ def close(self) -> None:
         self.reader.feed_eof()
         self.writer.close()
 
+    def is_closing(self) -> bool:
```

Comment:
Just realized I already made the mistake that this pattern causes (at least for me).

I have grown to dislike functions that return booleans.  As you pointed out in my PR, I incorrectly used this as a property rather than calling it.  Can we instead convert this to a `property`?

This way if we *incorrectly* try calling it we'll get an error, where-as, with this pattern, if we incorrectly treat it like a property, it silently always returns truthy.

Reply:
I've updated this in #1018 so if we want to just merge that as the union of these two I'm good with that.

Reply:
I implemented it as a method to match the underlying one, and because I intended to get rid of this and use `StreamWriter.is_closing()` directly once we migrate to 3.7 (the version in which it was introduced), but I guess that will be a while and yeah, I agree this makes more sense as a property, so I'll change it